src/helm/benchmark/run_specs.py

```diff
@@ -2415,18 +2443,26 @@ def get_cleva_spec(
         metric_specs = get_exact_match_metric_specs()
         if task in ["fact_checking", "bias"]:
             metric_specs += get_cleva_classification_metric_specs()
-    # elif task in ["conceptual_generalization"]:
-    #     adapter_spec = get_generation_adapter_spec(
-    #         instructions=prompt_setting.instructions,
-    #         input_noun=prompt_setting.input_noun,
-    #         newline_after_input_noun=prompt_setting.newline_after_input_noun,
-    #         output_noun=prompt_setting.output_noun,
-    #         newline_after_output_noun=prompt_setting.newline_after_output_noun,
-    #         max_train_instances=20,  # limited by the context length
-    #         max_tokens=10,
-    #     )
-    #     metric_specs = get_cleva_topk_accuracy_metric_specs() + get_cleva_generative_harms_metric_specs()
-    elif task in ["opinion_mining", "paraphrase_generation"]:
+    elif task in ["copyright"]:
+        adapter_spec = get_completion_adapter_spec(
+            temperature=inference_parameters.get("temperature", 0.2),
+            max_tokens=inference_parameters.get("max_tokens", 1024),
+            num_outputs=inference_parameters.get("num_outputs", 1),
+        )
+        args = {"normalize_by_prefix_length": True, "normalize_newline_space_tab": False}
+        metric_specs = get_cleva_copyright_metric_spec(args) + get_generative_harms_metric_specs()
```

Comment:
Use `get_cleva_generative_harms_metric_specs()` instead of `get_generative_harms_metric_specs()`.

Reply:
Corrected.